relnote(106) Add details of HTMLMetaElement.media DOM API (#21659)
mdn/content: files/en-us/mozilla/firefox/releases/106/index.md

#### DOM

> - The [`HTMLMetaElement.media`](/en-US/docs/Web/API/HTMLMetaElement/media) property is now supported which enables setting different theme colors based on `media` values (e.g. `max-width: 600px`).
>   Meta elements with `media` properties allows the browser to use the `content` value in conjunction with `theme-color` to set the page or UI colors for a given media query ({{bug(1706179)}}).

Reviewer: ```suggestion
- The [`HTMLMetaElement.media`](/en-US/docs/Web/API/HTMLMetaElement/media) property is now supported. This property enables you to set different theme colors based on `media` values (e.g. `max-width: 600px`).
  Meta elements with `media` properties allow the browser to use the `content` value in conjunction with `theme-color` to set the page or UI colors for a given media query ({{bug(1706179)}}).
```
